features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Shared/ApiKey.jsx

```diff
@@ -61,7 +61,11 @@ const tokens = (props) => {
                 }
                 break;
             case 'timeout':
-                newRequest.timeout = currentTarget.value;
+                if (!isNaN(currentTarget.value)) {
```

Comment:
is this a validation? if so do we show an error when it is invalid ?

Reply:
removed the Nan validation and used the joi library to validate